lib/create-http-client.js

```diff
@@ -123,6 +123,6 @@ export default function createHttpClient (axios, options) {
       ...newParams
     })
   }
-  rateLimit(instance)
+  rateLimit(instance, axiosOptions)
```

Comment:
do we still need to send this when we create the axios instance above? line 107

Reply:
Yes we still need I think since axios didn't merge my PR :(